include/trusted-firmware-a.mk

```diff
@@ -81,7 +81,7 @@ define Build/Compile/Trusted-Firmware-A
 		$(if $(DTC),DTC="$(DTC)") \
 		PLAT=$(PLAT) \
 		BUILD_STRING="OpenWrt v$(PKG_VERSION)-$(PKG_RELEASE) ($(VARIANT))" \
-		$(if $(CONFIG_BINUTILS_VERSION_2_39),LDFLAGS="-no-warn-rwx-segments") \
+		$(if $(CONFIG_BINUTILS_VERSION_2_39),$(CONFIG_BINUTILS_VERSION_2_40),LDFLAGS="-no-warn-rwx-segments") \
```

Comment:
I think Ansuel meant to negate that check so we don't have to extend it again for newer versions in the future.
Untested, but something like `$(if $(CONFIG_BINUTILS_VERSION_2_37)$(CONFIG_BINUTILS_VERSION_2_38),,LDFLAGS="-no-warn-rwx-segments")`

Reply:
It looks fine. I will test it with the `binutils-2.40-test` branch

Reply:
Changed that. Everything worked for me, thanks.